packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/README.md

```diff
@@ -5,23 +5,31 @@
 | Path | Contains |
 | ------------ | ------------ |
 | `lib/base/` | Common code used on more than one **feature** in the project. |
+| `lib/base/app/` | The root of the application and Environment configuration. |
 | `lib/base/common_blocs/` | Global [BLoCs][rx_bloc_info_lnk]|
 | `lib/base/common_ui_components/` | Reusable widgets (buttons, controls etc) |
 | `lib/base/common_use_cases/` | Global UseCases |
+| `lib/base/data_sources/` | All data sources are placed here. |
+| `lib/base/data_sources/domain_data_sources/` | Data sources,  relating to a specific domain |
```

Comment:
@RomanovaPrime, @DDavidPrime what about avoiding the redundancy of having `data_sources ` twice.
My suggestion is to rename 
- `data_sources/domain_data_sources` to `data_sources/domain/`
- `data_sources/local_data_sources` to `data_sources/local/`
- `data_sources/remote_data_sources` to `data_sources/remote/`

The file path would look like:
`lib/base/data_sources/domain/counter/counter_remote_data_source.dart` with a class name `CounterRemoteDataSource`

Reply:
I do agree for directories, but what about leave file and class names like they are? 
For example count_data_source.dart

Reply:
Since in Dart we have no namespaces I would vote for full name of the file as suggested above